lib/python/pyflyby/_autoimp.py

```diff
@@ -1489,6 +1489,14 @@ def get_known_import(fullname, db=None):
             logger.debug("get_known_import(%r): no known import for %r", fullname, partial_name)
             pass
     logger.debug("get_known_import(%r): found nothing", fullname)
+    """
+    # Write unknown imports to file which can later be removed as encountered
+    target_filename = Filename(target_filename or ".")
+    if target_filename.startswith("/dev"):
+        target_filename = Filename(".")
+    target_dirname = target_filename
+    write_file(target_dirname+"/unknown.py", fullname)
+    """
```

Comment:
What is the purpose of this? Should it be a comment?

Reply:
This is for future use, `unknown` would hold all unknown import package names (`fullname`).
TODO: Once a package becomes known in future that name can be removed from the file.

Reply:
Hmm maybe you could remove it from this PR then?

Reply:
Actually that was the part I wanted to implement, maybe I shall add a TODO with what else to be done over it?
Or do you think it doesn't make sense?

Reply:
I think it doesn't make sense to merge in code that has no execution pathway.  It also seems separate from the readme improvements, so I would recommend putting these in as separate PRs

Reply:
Thank you @animenon for the PR.  

A note to @scopatz : We had a programming contest around pyflyby in PyCon India 2020.  The idea there was to come up with a design (and implementation too) to add imports to known imports.  Well, this PR is supposed to go there.  That said, I agree with you, @scopatz .  There is no point of a code path which never gets executed. 

@animenon, as for the contest, we are not accepting this PR since it does not implement what was asked. However, you can still continue to collaborate (README related changes) if you want.

Reply:
Sure @return007, wasn't intending it to be part of the contest.
Don't think anyone was able to figure out the repo structure.